src/three-components/CachingGLTFLoader.js

```diff
@@ -35,9 +36,10 @@ export class CachingGLTFLoader {
       cache.set(url, loadWithLoader(url, this.loader));
     }
 
-    const gltf = await cache.get(url);
+    const gltf = cloneGltf(await cache.get(url));
+    const model = gltf.scene ? gltf.scene : null;
```

Comment:
Wouldn't this turn into `null.userData.animations =` a couple of lines down?

Reply:
👍 Will fix